### PR TITLE
(PUP-5445) Add Arista support

### PR DESCRIPTION
**Pull request by LuvCurves**

### CODE REVIEW

File: acceptance/lib/puppet/acceptance/install_utils.rb [hunk 1]
```diff
@@ -13,6 +13,7 @@ module InstallUtils
         :solaris_10    => /solaris-10/,
         :solaris_11    => /solaris-11/,
         :windows       => /windows/,
+        :eos           => /^eos-/,
```
Comment (kylog) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: Thanks for fixing this one up @LuvCurves. Can you apply the same to *all* the regexes in this PR? I have the same concern for the skips and confines.

Comment (LuvCurves) [inline acceptance/lib/puppet/acceptance/install_utils.rb]: Done

**Comment by MikaelSmith** [comment]
:+1:

**Comment by MikaelSmith** [comment]
Does this require a beaker change? If I try to run it, I get `#<RuntimeError: No repository installation step for eos yet...>.

**Comment by LuvCurves** [comment]
@MikaelSmith Yes, This requires what will be Beaker 2.28.0

**Comment by kylog** [comment]
@LuvCurves once 2.28 is released, can you extend this PR to bump the minimum beaker requirement at https://github.com/puppetlabs/puppet/blob/master/acceptance/Gemfile#L15?

**Comment by LuvCurves** [comment]
@kylog Will do.

**Comment by LuvCurves** [comment]
Beaker 2.28.0 is available.

**Comment by MikaelSmith** [comment]
:+1:

**Comment by MikaelSmith** [comment]
Waiting on CI to be green again.

**Comment by joshcooper** [comment]
If this is required for 4.3.0, please retarget for stable

**Comment by LuvCurves** [comment]
Closing and retargeting to stable

**Pull request closed without merging**